### ISSUE TITLE
List of Projects Required for LWKD

### ISSUE THREAD

**Issue by fykaa** [kubernetes-sigs/lwkd#232]
We require a list of relevant projects from which we can draw news, updates, PRs, etc. for the LWKD newsletter.

Also feel free to include them in the comments, and we will take them into account.

**Comment by jberkus** [comment]
Yeah, so we want to limit this to close projects -- ones that are required to build Kubernetes, mostly, and things like installers.  We would NOT want to include the whole CNCF universe.

Examples would be:

* Etcd
* CSI
* CNI
* CRI-O & Containerd
* Kops
* CoreDNS
* gRPC
* GoLang (this usually shows up in Version Updates anyway)

And, of course, subprojects of Kubernetes are all included, like:

* kubeadm
* clusterAPI
* nginx-ingress

There's some borderline ones, where we might include something from them if it's *major*, like new major version or a serious security hole.  This could include:

* Helm
* Prometheus
* OpenTelemetry
* Sigstore (because it affects our builds)

A good rule of thumb would be: "if the project exists in a competitive space, probably don't include it".

**Comment by jberkus** [comment]
per other issue, we've added Kubespray.
